src/components/CheckEditor/CheckEditor.tsx

```diff
@@ -205,24 +189,22 @@ export const CheckEditor = ({ onReturn, checks }: Props) => {
                 Cancel
               </LinkButton>
             </HorizontalGroup>
-            {submissionError && (
-              <div className={styles.submissionError}>
-                <Alert title="Save failed" severity="error">
-                  {`${submissionError.status}: ${
-                    submissionError.data?.msg?.concat(', ', submissionError.data?.err ?? '') ?? 'Something went wrong'
-                  }`}
-                </Alert>
-              </div>
-            )}
           </form>
         </FormProvider>
       </>
+      {error && (
+        <div className={styles.submissionError}>
+          <Alert title="Save failed" severity="error">
+            {error.message ?? 'Something went wrong'}
+          </Alert>
+        </div>
+      )}
```

Comment:
I thought some of the alerts were displaying in strange places so I am trying to standardise them so they appear after all the submit buttons.